README.md

```diff
@@ -121,13 +107,56 @@ OPTIONS
   --loglevel=(trace|debug|info|warn|error|fatal|TRACE|DEBUG|INFO|WARN|ERROR|FATAL)  [default: warn] logging level for
                                                                                     this command invocation
 
+  --verbose                                                                         emit additional command output to
+                                                                                    stdout
+
+DESCRIPTION
+  Output includes your access token, client Id, connected status, org ID, instance URL, username, and alias, if
+  applicable.
+  Use --verbose to include the SFDX auth URL.
+  Including --verbose displays the sfdxAuthUrl property only if you authenticated to the org using auth:web:login (not
+  auth:jwt:grant)
+
 EXAMPLES
-  $ sfdx hello:org --targetusername myOrg@example.com --targetdevhubusername devhub@org.com
-     Hello world! This is org: MyOrg and I will be around until Tue Mar 20 2018!
-     My hub org id is: 00Dxx000000001234
+  sfdx force:org:display
+  sfdx force:org:display -u me@my.org
+  sfdx force:org:display -u TestOrg1 --json
+  sfdx force:org:display -u TestOrg1 --json > tmp/MyOrgDesc.json
+```
+
+_See code: [src/commands/force/org/display.ts](https://github.com/salesforcecli/plugin-org/blob/v1.1.1/src/commands/force/org/display.ts)_
+
+## `sfdx force:org:list [--all] [-p --clean] [--skipconnectionstatus] [--verbose] [--json] [--loglevel trace|debug|info|warn|error|fatal|TRACE|DEBUG|INFO|WARN|ERROR|FATAL]`
+
+list all orgs that the Salesforce CLI has created or authenticated to
+
+```
+USAGE
+  $ sfdx force:org:list [--all] [-p --clean] [--skipconnectionstatus] [--verbose] [--json] [--loglevel
+  trace|debug|info|warn|error|fatal|TRACE|DEBUG|INFO|WARN|ERROR|FATAL]
+
+OPTIONS
+  -p, --noprompt                                                                    Do not prompt for confirmation.
 
-  $ sfdx hello:org --name myname --targetusername myOrg@example.com
-     Hello myname! This is org: MyOrg and I will be around until Tue Mar 20 2018!
+  --all                                                                             Lists all authenticated orgs,
```

Comment:
Per guidelines, flags start with imperative verb:

"list all authenticated orgs, including expired, deleted, and unknown-status scratch orgs"